packages/ses/test/test-anticipate-async-iterator-helpers-shimmed.js

```diff
@@ -0,0 +1,38 @@
+// KLUDGE HAZARD The core-js shims are written as sloppy code
+// and so introduce sloppy functions.
+import 'core-js/actual/async-iterator/index.js';
+import test from 'ava';
+import '../index.js';
+
+// KLUDGE HAZARD only for testing with the sloppy modules of the
+// core-js iterator shim.
+// We mutate the permits to tolerates the sloppy functions for testing
+// by sacrificing security. The caller and arguments properties of
+// sloppy functions violate ocap encapsulation rules.
+import { AsyncFunctionInstance } from '../src/permits.js';
+
+AsyncFunctionInstance.arguments = {};
+AsyncFunctionInstance.caller = {};
+
+// Skipped because the core-js shim seems to miss the
+// actual %AsyncIteratorPrototype%,
+// so it creates a new one, causing us to fail because lockdown correctly
+// detects the conflicting definitions.
+// TODO report the bug to core-js
```

Comment:
Let's resolve this TODO before merging, and include a link to the issue?

Reply:
Done! At https://github.com/zloirock/core-js/issues/1267

Reply:
Context on core-js and its single maintainer https://github.com/zloirock/core-js/blob/master/docs/2023-02-14-so-whats-next.md 
TL;DR: all attempts to sustain himself while working on core-js full time have been unsuccessful. He might stop at any point and he's in trouble. 
I checked and he does seem active, so it's possible the last attempt proposed in the doc is working for now.